src/ContentPage.jsx

```diff
@@ -62,16 +62,17 @@ const Video = (props) => (
 
 const AddVideoModal = (props) => (
   <div>
-    <button type="button" className="btn btn-primary" data-toggle="modal" data-target="#AddVideoModal">Add Video</button>
-
+    <div id="buttonOnRight">
```

Comment:
I like when things are right

Reply:
This comment LEFT me feeling RIGHT on the inside